extension/api.js

```diff
@@ -41,6 +41,7 @@ const PROXY_CONFIG_TYPES = [
   "direct",
   "manual",
   "pac",
+  "unused", // nsIProtocolProxyService.idl skips index 3.
```

Comment:
Would this increase the difficulty of the analysis?

If so, how about appending "system_real", and then documenting that "system" is actually "wpad"? The string is technically incorrect, but the results may be easier to interpret.

Reply:
No, right now system shows up as "unknown" which in the code was just a safety catch and should never happen.  wpad shows up as system, but the numbers are minuscule and will just be a touch of noise.  those with system set may be interesting but are not anything we will do anything about.